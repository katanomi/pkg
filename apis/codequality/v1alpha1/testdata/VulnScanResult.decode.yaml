result: Succeeded
targets[0].uri: aquasec/trivy:latest
targets[0].type: Image
targets[0].cvss.source: Nvd
targets[0].cvss.severity: Critical
targets[0].cvss.score: "7.3"
targets[0].criticalCount: "10"
targets[0].highCount: "10"
targets[0].mediumCount: "10"
targets[0].lowCount: "10"
targets[0].unknownCount: "10"
targets[1].uri: ubuntu:22.04@sha265:aaaaaaaaaaaaaa
targets[1].type: Image
targets[1].cvss.source: Nvd
targets[1].cvss.severity: Critical
targets[1].cvss.score: "7.3"
targets[1].criticalCount: "10"
targets[1].highCount: "10"
targets[1].mediumCount: "10"
targets[1].lowCount: "10"
targets[1].unknownCount: "10"