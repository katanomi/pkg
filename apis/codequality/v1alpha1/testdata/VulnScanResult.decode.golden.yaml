targets:
  - uri: aquasec/trivy:latest
    highCount: 10
    lowCount: 10
    mediumCount: 10
    unknownCount: 10
    criticalCount: 10
    cvss:
      source: Nvd
      severity: Critical
      score: 7.3
    type: Image
  - highCount: 10
    type: Image
    mediumCount: 10
    criticalCount: 10
    uri: ubuntu:22.04@sha265:aaaaaaaaaaaaaa
    lowCount: 10
    unknownCount: 10
    cvss:
      source: Nvd
      score: 7.3
      severity: Critical
result: Succeeded